demo/README.md

```diff
@@ -2,45 +2,144 @@
-- full implementation of OTel in the backend
+The app is built using:
+
+- Client
+  - [React](https://reactjs.org/)
+  - [React Router](https://reactrouter.com/en/main)
+  - [Redux](https://redux.js.org/)
+  - [Redux Toolkit](https://redux-toolkit.js.org/)
+  - [Redux Toolkit Query](https://redux-toolkit.js.org/rtk-query/overview)
+  - [Grafana JavaScript Agent](https://github.com/grafana/grafana-javascript-agent)
+  - [Server-side rendering (SSR)](https://reactjs.org/docs/react-dom-server.html)
+  - [React Bootstrap](https://react-bootstrap.github.io/)
+  - [React Hook Form](https://react-hook-form.com/)
+- Server
+  - [Express](https://expressjs.com/)
+  - [PostgreSQL](https://www.postgresql.org/)
+  - [Sequelize](https://sequelize.org/)
+  - [OpenTelemetry](https://opentelemetry.io/docs/instrumentation/js/)
+    - Instrumentations:
+      - [Express](https://www.npmjs.com/package/@opentelemetry/instrumentation-express)
+      - [HTTP](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)
+      - [PostgreSQL](https://www.npmjs.com/package/@opentelemetry/instrumentation-pg)
+      - [Winston](https://www.npmjs.com/package/@opentelemetry/instrumentation-winston)
+  - [Winston](https://github.com/winstonjs/winston)
+  - [prom-client](https://github.com/siimon/prom-client)
+- Misc
+  - [dotenv](https://github.com/motdotla/dotenv)
+  - [Vite](https://vitejs.dev/)
+
+Besides the app itself, the demo can also spawn the entire architecture for testing Grafana JavaScript Agent:
+
+- [Grafana](https://grafana.com/oss/grafana/)
+- [Loki](https://grafana.com/oss/loki/)
+- [Tempo](https://grafana.com/oss/tempo/)
+- [Cortex](https://cortexmetrics.io/)
 
 ## Installation
 
-1. Clone the repository:
-
-   ```shell
-   git clone git@github.com:grafana/grafana-javascript-agent.git
-   ```
-
-2. Install the dependencies:
-
-   ```shell
-   yarn
-   ```
-
-   Running the above command in the root directory of the repository will install all the necessary dependencies for
-   the packages and the demo.
-
-3. Run the infrastructure:
-
-   ```shell
-   docker-compose up -d
-   ```
-
-4. Run the demo:
-
-   ```shell
-   // if you want to run the demo locally
-   yarn start
-
-   // Alternatively you can run the demo using Docker
-   docker compose --profile demo up -d
-   ```
-
-## Journey
-
-TODO
+The demo can be run using containers. This is the easiest way to get started as it will spin up all the necessary
+services. Moreover, the source code is mapped in the container so everytime a change occurs, there's no need to restart
+the container(s).
+
+The app will be available at [http://localhost:5173](http://localhost:5173) and Grafana will start at [http://localhost:3000/](http://localhost:3000/).
+
+To start everything up, simply run:
+
+```shell
+docker-compose --profile demo up -d
+```
+
+This will automatically install dependencies, build the demo and start it in `development` mode.
+
+## Presentation
```

Comment:
Not sure if presentation is clear here. Maybe something like instrumentation or data collection?